config/tipoff.php

```diff
@@ -4,97 +4,97 @@
 
     'model_class' => [
 
-        'block' => \App\Models\Block::class,
+        'block' => \Tipoff\Scheduling\Models\Block::class,
 
-        'booking' => \App\Models\Booking::class,
+        'booking' => \Tipoff\Bookings\Models\Booking::class,
 
-        'cart' => \App\Models\Cart::class,
+        'cart' => \Tipoff\Checkout\Models\Cart::class,
 
-        'cart_item' => \App\Models\CartItem::class,
+        'cart_item' => \Tipoff\Checkout\Models\CartItem::class,
 
-        'competitor' => \App\Models\Competitor::class,
+        'competitor' => \Tipoff\Reviews\Models\Competitor::class,
 
-        'contact' => \App\Models\Contact::class,
+        'contact' => \Tipoff\Forms\Models\Contact::class,
 
-        'customer' => \App\Models\Customer::class,
+        'customer' => \Tipoff\Addresses\Models\Customer::class, // Will be renamed later to address when new features added
 
-        'discount' => \App\Models\Discount::class,
+        'discount' => \Tipoff\Discounts\Models\Discount::class,
 
-        'fee' => \App\Models\Fee::class,
+        'fee' => \Tipoff\Fees\Models\Fee::class,
 
-        'feedback' => \App\Models\Feedback::class,
+        'feedback' => \Tipoff\Feedback\Models\Feedback::class,
 
-        'flex_day' => \App\Models\FlexDay::class,
+        'flex_day' => \Tipoff\FlexScheduling\Models\FlexDay::class,
 
-        'game' => \App\Models\Game::class,
+        'game' => \Tipoff\Scheduling\Models\Game::class,
 
-        'hold' => \App\Models\Hold::class,
+        'hold' => \Tipoff\Scheduling\Models\Hold::class,
 
         'image' => \DrewRoberts\Media\Models\Image::class,
 
-        'insight' => \App\Models\Insight::class,
+        'insight' => \Tipoff\Reviews\Models\Insight::class,
 
-        'invoice' => \App\Models\Invoice::class,
+        'invoice' => \Tipoff\Invoices\Models\Invoice::class,
 
-        'key' => \App\Models\Key::class,
+        'key' => \Tipoff\Reviews\Models\Key::class,
 
-        'location' => \App\Models\Location::class,
+        'location' => \Tipoff\Locations\Models\Location::class,
 
-        'market' => \App\Models\Market::class,
+        'market' => \Tipoff\Locations\Models\Market::class,
 
-        'note' => \App\Models\Note::class,
+        'note' => \Tipoff\Notes\Models\Note::class,
 
-        'order' => \App\Models\Order::class,
+        'order' => \Tipoff\Checkout\Models\Order::class,
+
+        'order_item' => \Tipoff\Checkout\Models\OrderItem::class,
 
         'page' => \DrewRoberts\Blog\Models\Page::class,
 
         'participant' => \App\Models\Participant::class,
```

Comment:
I'm still undecided about where to put the participant model. It makes sense in tipoff/escape-room but it is more closely related to the games model which is in tipoff/scheduling or the signature model in tipoff/waivers and I may also pull it out into the tipoff/remote-monitoring package. I will get back on this over the weekend. Let me sleep on it.

Reply:
I decided to move it to the tipoff/escape-room package. This will allow the other 3 packages to use it, but a Laravel project won't be required to have any of those 3 additional packages. All 3 of those will require the escape-room package.